spinta/auth.py

```diff
@@ -472,18 +473,24 @@ def get_scope_name(
     elif isinstance(node, Model):
         name = node.model_type()
     elif isinstance(node, Property):
-        name = node.model.model_type() + "_" + node.place
+        name = node.model.model_type() + "_" + node.place if not udts else node.model.model_type() + "/@" + node.place
```

Comment:
We should try to avoid using negatives if possible:
```suggestion
        name = node.model.model_type() + "/@" + node.place if udts else node.model.model_type() + "_" + node.place
```

Reply:
Changed, I made it this way to keep the sequence of choices the same. So it was `name = node.model.model_type() + "_" + node.place` and then I added the new statement at the end. But this is better